electrolyzer/cell.py

```diff
@@ -61,10 +62,11 @@ def calc_open_circuit_voltage(self, temperature):
         T_K = convert_temperature([temperature], "C", "K")[0]
         E_rev_0 = self.calc_reversible_voltage()
         p_anode = P_ATMO  # (Pa) assumed atmo
-        p_cathode = P_ATMO
+        p_cathode = 30 * P_STD  # (Pa) 30 bars
```

Comment:
do you think we might eventually expose these pressure values as config variables for users?

Reply:
I thought about awhile back and it may be worth submitting it as a suggested upgrade. I told myself I'd come back to it when I better understand the model_schema format. It'll need some extra pieces to default to a certain value and check to make sure the user inputs are valid. For now, I think just leaving it with atmo on one side, and 30bar on the other to be consistent with customer requests.